src/components/Vote/Proposals.tsx

```diff
@@ -31,6 +31,8 @@ const ProposalsWrapper = styled.div`
       background-color: var(--color-gold);
       border: none;
       color: #fff;
+      width: 150px;
+      height: 40px;
```

Comment:
We should use the the `spacing` value from the theme to calculate those.

Reply:
I don't know that is particularly useful in this case. I understand spacing to be a multiplier on a value to create space between elements. The benefit is you could modify the base value and the app should nicely increase or decrease spaces between elements proportionally. I don't think we want to mix the size of elements with that

Reply:
The way I've used the base spacing in the past was to make sure all the elements of the UI used a common base to define anything related to sizes and spacings, so that the UI flows nicely and you don't get odd values.
I don't think it can be used for responsiveness anyway, even if it is only used for spacings, as sizes would not update accordingly and it would quickly look odd.